pages/500.tsx

```diff
@@ -0,0 +1,19 @@
+import {
```

Comment:
Look into the _error class instead to handle all error codes: https://nextjs.org/docs/advanced-features/custom-error-page#more-advanced-error-page-customizing.